include/openssl/dh.h

```diff
@@ -134,14 +138,18 @@ OPENSSL_EXPORT int DH_set0_pqg(DH *dh, BIGNUM *p, BIGNUM *q, BIGNUM *g);
 // |DH_generate_key| will use the bit length of p.
 OPENSSL_EXPORT int DH_set_length(DH *dh, unsigned priv_length);
 
-
```

Comment:
I think the double new line might be intentional for `docs.go` to know this is a separate section